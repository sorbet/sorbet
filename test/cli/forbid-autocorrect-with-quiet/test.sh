#!/bin/sh
exec main/sorbet --silence-dev-message --quiet --autocorrect input.rb 2>&1
