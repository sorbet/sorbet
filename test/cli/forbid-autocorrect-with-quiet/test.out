You may not use autocorrect when silencing errors.
